src/linked_list.py

```diff
@@ -0,0 +1,59 @@
+class Node:
+    def __init__(self, val, next_node):
+        self.val = val
+        self.next_node = next_node
+
+    def __repr__(self):
+        return f"({self.val})->{self.next_node}"
+
+
+class LinkedList:
+    def __init__(self, iterable=None):
```

Comment:
If you're using `typing` here still, you would do: `iterable: Optional[List[Any]] = None`. 

`Optional` because it can be None, and `List[Any]` meaning it can be a list containing any data type. :)